tool_ips2crash/ips2crash/IPSReport+CrashRepresentation.m

```diff
@@ -297,9 +297,11 @@ - (NSString *)crashTextualRepresentation
         
         NSString * tCrashedString=(bThread.triggered==YES) ? @" Crashed":@"";
         
+        NSString * tNameString=(bThread.name!=nil) ? [NSString stringWithFormat:@" %@",bThread.name] : @"";
+
         NSString * tDispatchQueueString=(bThread.queue!=nil) ? [NSString stringWithFormat:@": Dispatch queue: %@",bThread.queue] : @"";
         
-        [tMutableString appendFormat:@"Thread %lu%@:%@\n",(unsigned long)bThreadIndex,tCrashedString,tDispatchQueueString];
+        [tMutableString appendFormat:@"Thread %lu%@:%@%@\n",(unsigned long)bThreadIndex,tCrashedString, tNameString, tDispatchQueueString];
```

Comment:
I don't think you can have both a thread name and a dispatch queue name.
Do you have an example of a .crash or .ips file where there are both a name and a dispatch queue.

Reply:
Please have a look at this .ips file:
[REAPER-2022-02-11-094340.ips.txt](https://github.com/packagesdev/ips2crash/files/8047201/REAPER-2022-02-11-094340.ips.txt)

Opening this file on a Big Sur system produces this content:
`Thread 0 Crashed:: reaper Dispatch queue: com.apple.main-thread`

Here `reaper` is the thread name.

This line has been generated from this part of the .ips file:
`"name":"reaper","queue":"com.apple.main-thread","frames":[...`
 
 So I assume we can have both a name and a queue to qualify a thread.
 
 Please let me know, if you expect anything else for this contribution.

Reply:
Great ! Thank you for merging this contribution. 
I reviewed the extra patch you added to match the old crash format and it looks good to me. Thanks.